loader.qml

```diff
@@ -1,57 +1,193 @@
 import QtQuick 2.15
 import QtGraphicalEffects 1.12
 import QtQuick.Layouts 1.2
-import QtQuick.Controls 1.4
+import QtQuick.Controls 2.15
 import QtQuick.Dialogs 1.1
 
 ApplicationWindow {
     id: root
+    visible: true
+    minimumWidth: 640
+    minimumHeight: mysize + header.height // + menuBar.height
     property bool displayAmbient: false
     property bool round: true
+    property int mysize: 640
     property var testface: Qt.application.arguments[1]
+    readonly property var mouseWheelScale: 1/15
 
-    toolBar: ToolBar {
-        RowLayout {
-            anchors.fill: parent
-            anchors.margins: spacing
-            CheckBox {
-                id: roundCheckBox
-                text: "Round"
-                Component.onCompleted: checked = round
-                onCheckedChanged: round = checked; 
-            }
-            CheckBox {
-                id: checkBox12h
-                text: "12h time"
-                Component.onCompleted: checked = use12H.value
-                onCheckedChanged: use12H.value = checked; 
-            }
-            CheckBox {
-                id: ambientCheckBox
-                text: "Dark"
-                Component.onCompleted: checked = displayAmbient
-                onCheckedChanged: displayAmbient = checked; 
+        /*
+    menuBar: MenuBar {
+        Menu {
+            title: qsTr("&Help")
+            Action { text: qsTr("&About") }
+        }
+    }
+        */
+    header: ToolBar {
+        background: Rectangle {
+            color: "lightblue"
+        }
+        ColumnLayout {
+            RowLayout {
+                Layout.alignment: Qt.AlignCenter
+                CheckBox {
+                    id: roundCheckBox
+                    text: "Round"
+                    Component.onCompleted: checked = round
+                    onCheckedChanged: round = checked; 
+                }
+                CheckBox {
+                    id: halfsize
+                    text: "320x320"
+                    Component.onCompleted: checked = mysize == 320
```

Comment:
It's a good practice to add parenthesis around complex expressions to make it easier to read. For example checked = (mySize == 320)
It makes the code a little bit more readable, same for next line